Formula/goplus.rb

```diff
@@ -1,8 +1,9 @@
 class Goplus < Formula
   desc "Programming language for engineering, STEM education, and data science"
   homepage "https://goplus.org"
-  url "https://github.com/goplus/gop/archive/v1.0.37.tar.gz"
-  sha256 "edbb7eb99f175423d583de156967b5d9ae632b2fe405b6c1a36ede47591d7dd5"
+  url "https://github.com/goplus/gop.git",
+      tag:      "v1.0.38",
+      revision: "8ab2d87ef7948cf9fb66b3edeeb7caa1320db47c"
```

Comment:
Can we just create the `VERSION` file instead of using a Git clone?

Reply:
I looked at the source file, I dont think it does respect `VERSION` file.

Reply:
ok, let me give a shot.

Reply:
I did not get the tarball version working. :(

Receving test failure as below
```
$ /opt/homebrew/Cellar/goplus/1.0.38/bin/gop env GOPVERSION
panic: Invalid buildVersion: 1.0.38

goroutine 1 [running]:
github.com/goplus/gop/env.init.0()
	/private/tmp/goplus-20211228-27233-ign710/gop-1.0.38/env/version.go:35 +0xd4
```

Reply:
maybe just missing a leading `v`? this worked for me on intel

```ruby
(buildpath/"VERSION").write "v#{version}"
```